An error occur when number of rows are entered

When number of rows is entered in the terminal the following error occurs:

`e
Exception in thread "main" Syntax error compiling at (/private/var/folders/w3/0vsvl18n68x_lmyvg28l3lbshc3g7f/T/form-init6570231961523131482.clj:1:125).
        at clojure.lang.Compiler.load(Compiler.java:7647)
        at clojure.lang.Compiler.loadFile(Compiler.java:7573)
        at clojure.main$load_script.invokeStatic(main.clj:452)
        at clojure.main$init_opt.invokeStatic(main.clj:454)
        at clojure.main$init_opt.invoke(main.clj:454)
        at clojure.main$initialize.invokeStatic(main.clj:485)
        at clojure.main$null_opt.invokeStatic(main.clj:519)
        at clojure.main$null_opt.invoke(main.clj:516)
        at clojure.main$main.invokeStatic(main.clj:598)
        at clojure.main$main.doInvoke(main.clj:561)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.lang.Var.applyTo(Var.java:705)
        at clojure.main.main(main.java:37)
Caused by: java.lang.IllegalStateException: Attempting to call unbound fn: #'pegthing.core/prompt-move
        at clojure.lang.Var$Unbound.throwArity(Var.java:45)
        at clojure.lang.AFn.invoke(AFn.java:32)
        at pegthing.core$prompt_empty_peg.invokeStatic(core.clj:235)
        at pegthing.core$prompt_empty_peg.invoke(core.clj:229)
        at pegthing.core$prompt_rows.invokeStatic(core.clj:243)
        at pegthing.core$prompt_rows.invoke(core.clj:237)
        at pegthing.core$_main.invokeStatic(core.clj:262)
        at pegthing.core$_main.doInvoke(core.clj:259)
        at clojure.lang.RestFn.invoke(RestFn.java:397)
        at clojure.lang.Var.invoke(Var.java:380)
        at user$eval140.invokeStatic(form-init6570231961523131482.clj:1)
        at user$eval140.invoke(form-init6570231961523131482.clj:1)
        at clojure.lang.Compiler.eval(Compiler.java:7176)
        at clojure.lang.Compiler.eval(Compiler.java:7166)
        at clojure.lang.Compiler.load(Compiler.java:7635)
        ... 12 more`

--- a/src/pegthing/core.clj
+++ b/src/pegthing/core.clj
@@ -208,7 +208,7 @@
        default
        (clojure.string/lower-case input)))))
 
-(defn characters-as-string
+(defn characters-as-strings
   "Given a string, return a collection consisting of each individual character"
   [string]
   (re-seq #"[a-zA-Z]" string))
@@ -231,7 +231,7 @@
   (println "\nHere's your board:")
   (print-board board)
   (println "Move from where to where? Enter two letters:")
-  (let [input (map letter->pos (characters-as-string (get-input)))]
+  (let [input (map letter->pos (characters-as-strings (get-input)))]
     (if-let [new-board (make-move board (first input) (second input))]
       (user-entered-valid-move new-board)
       (user-entered-invalid-move board))))

--- a/src/pegthing/core.clj
+++ b/src/pegthing/core.clj
@@ -226,6 +226,16 @@
     (prompt-move board)
     (game-over board)))
 
+(defn prompt-move
+  [board]
+  (println "\nHere's your board:")
+  (print-board board)
+  (println "Move from where to where? Enter two letters:")
+  (let [input (map letter->pos (characters-as-string (get-input)))]
+    (if-let [new-board (make-move board (first input) (second input))]
+      (user-entered-valid-move new-board)
+      (user-entered-invalid-move board))))
+
 (defn prompt-empty-peg
   "Ask the position of the initial removed peg"
   [board]

